public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts

```diff
@@ -252,6 +254,14 @@ export function createDashboardSceneFromDashboardModel(oldModel: DashboardModel,
         reloadOnScopesChange: oldModel.meta.reloadOnScopesChange,
         uid: oldModel.uid,
       }),
+      new ReloadOnTimeRangeChangeBehavior({
+        reloadOnTimeRangeChange: oldModel.meta.reloadOnTimeRangeChange,
+        uid: oldModel.uid,
+      }),
+      new ReloadOnFiltersChangeBehavior({
+        reloadOnFiltersChange: oldModel.meta.reloadOnFiltersChange,
+        uid: oldModel.uid,
+      }),
```

Comment:
Think we should unify this under a single variable. Having two toggles controlling this and two behaviors makes the code veryb "scattered". What if we had a single toogle and behavior, and let the "proxy" decide whether or not the dashboard needs to be refreshed?

Reply:
yeah, maybe separate behaviors are fine. But having 3 toggles to controll similar thing - that should work together - is a bit overkill to me :)